src/Content/Text/BBCode.php

```diff
@@ -236,8 +236,11 @@ public static function toPlaintext(string $text, bool $keep_urls = true): string
 		// Remove attachment
 		$text = self::replaceAttachment($text);
 
-		$naked_text = HTML::toPlaintext(self::convert($text, false, 0, true), 0, !$keep_urls);
+		$naked_text = HTML::toPlaintext(self::convert($text, false, BBCode::EXTERNAL, true), 0, !$keep_urls);
```

Comment:
Can't this line be replaced by `self::toPlaintext`?

Reply:
No, this is a different function with the same name.

Reply:
What do you mean a different function with the same name?

Reply:
I'm slightly confused. Your suggestion is to call `BBCode::toPlaintext` instead of `HTML::toPlaintext` in the function `BBCode::toPlaintext`, right?

Reply:
Ooooh, sorry, I didn’t realize it was in `BBCode::toPlaintext’ 🤦‍♂️